docs/src/index.md

```diff
@@ -25,25 +25,46 @@ Given an observation ``y``, the computer model ``\mathcal{G}``, the observationa
  
 As the name suggests, `CalibrateEmulateSample.jl` breaks this problem into a sequence of three steps: calibration, emulation, and sampling. A comprehensive treatment of the calibrate-emulate-sample approach to Bayesian inverse problems can be found in [Cleary et al. (2020)](https://arxiv.org/pdf/2001.03689.pdf).
 
-### The three steps of the algorithm:
+### The three steps of the algorithm: see our walkthrough of the [Sinusoid Example](@ref)
 
-The **calibrate** step of the algorithm consists of an application of [Ensemble Kalman Processes](https://github.com/CliMA/EnsembleKalmanProcesses.jl), which generates input-output pairs ``\{\theta, \mathcal{G}(\theta)\}`` in high density around an optimal parameter ``\theta^*``. This ``\theta^*`` will be near a mode of the posterior distribution (Note: This the only time we interface with the forward model ``\mathcal{G}``).
+**Given some noisy observations...**
+```@raw html
+<img src="assets/sinusoid_true_vs_observed_signal.png" width="400">
+```
+
+1. The **calibrate** step of the algorithm consists of an application of [Ensemble Kalman Processes](https://github.com/CliMA/EnsembleKalmanProcesses.jl), which generates input-output pairs ``\{\theta, \mathcal{G}(\theta)\}`` in high density around an optimal parameter ``\theta^*``. This ``\theta^*`` will be near a mode of the posterior distribution (Note: This the only time we interface with the forward model ``\mathcal{G}``).
+
+**calibrate with EKP to generate data pairs...**
+```@raw html
+<img src="assets/sinusoid_eki_pairs.png" width="400">
+```
 
-The **emulate** step takes these pairs ``\{\theta, \mathcal{G}(\theta)\}`` and trains a statistical surrogate model (e.g., a Gaussian process), emulating the forward map ``\mathcal{G}``.
+2. The **emulate** step takes these pairs ``\{\theta, \mathcal{G}(\theta)\}`` and trains a statistical surrogate model (e.g., a Gaussian process), emulating the forward map ``\mathcal{G}``.
```

Comment:
in the built docs the 2. and 3. appear as 1s

Reply:
Removed numbering, as it seems markdown won't do custom bullets